doc/src/advanced_tutorial/add_move_minimal.md

```diff
@@ -0,0 +1,236 @@
+
+A simple translation of a molecule works as follows:
+1. randomly select a molecule of the system
+2. randomly translate the center of mass position of the molecule (which leads to a translation of all positions of the atoms of said molecule)
+3. compute the energy difference due to moving the molecule. The energy difference is needed to decide wheter a move is accepted or rejected.
+4. if the move is accepted, update the positions of the atoms in the molecule
+5. if the move is rejected, leave the positions untouched
+
+Because `McMove` is a trait, we have to generate a `struct` for our move that implements the `MCMove` trait. We first consider some fields in which we store the  information needed. Let's start with this:
+
+```rust
+/// Monte-Carlo move for translating a molecule
+pub struct Translate {
+    /// System state prior to the move
+    old_system: System,
+    /// Maximum displacement value
+    dr: f64,
+    /// Translation range for random number generation
+    range: Range<f64>,
+}
+```
+
+We store the current (old) system before changing it in `old_system`.
+`dr` is called the "maximum displacement". When deciding how far we move a molecule, we pick three random (uniformly distributed) values in the `range` [`-dr, +dr`] and create a translation vector from it.
+For larger `dr` values, a molecule moves further away from its initial position *on average*. 
+
+> About the maximum displacement:  
+> `dr` can be adjusted during the course of a simulation to increase efficiency.
+> To illustrate this, think of a system containing densely packed hard spheres.
+> Hard spheres are spheres that do not interact with each other but are not allowed to overlap.
+> If we pick a sphere and translate it only a tiny bit, chances are good that it will not overlap with its neighbours.
+> Accordingly, this new configuration is accepted.
+> In contrast, if `dr` is large (and therefore the average displacement), it is very likely that the new configuration leads to an overlap and the proposed configuration is rejected.
+> Now, if we set `dr` to a large value and never change it, our move produces
+configurations that are rejected almost exclusively.
+> To efficiently sample configurations it is convenient to change `dr` during the simulation based on the ratio of accepted and rejected moves.
+> We will come back to this issue in our refined implementation.
+
+We have our struct, now let's write a function to create an instance of it. In rust, it is idiomatic to name this function `new`.
+
+```rust
+// File: translate.rs
+impl Translate {
+    fn new(dr: f64) -> Translate {
+        Translate {
+            old_system: System::default(),
+            dr: dr,
+            range: Range::new(-dr, dr),
+        }
+    }
+}
+```
+
+The `new` function takes a single argument, the maximum displacement `dr`.
+The internal unit of length - the displacement is a length - is Angstrom.
```

Comment:
We don't need to specify the internal unit. We can just say that all lengths are guaranteed to be in internal units, and maybe link to the internal unit definition.